statistics/handle/update_test.go

```diff
@@ -741,7 +741,7 @@ func (s *testStatsSuite) TestQueryFeedback(c *C) {
 	}{
 		{
 			// test primary key feedback
-			sql: "select * from t where t.a <= 5",
+			sql: "select * from t where t.a <= 5 order by a desc",
```

Comment:
why changing this?

Reply:
To cover the case of order by desc.

Reply:
can we keep the old test and use another new test to cover the case of `order by desc`?

Reply:
It needs a new environment since the stats are already changed, and I don't think it is necessary since the old case is already tested by many others.